src/components/indicator/Indicator.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../styles/variables';
+
+$indicator-max-width: 584px;
+
+.bdl-Indicator {
+    position: fixed;
+    bottom: 0;
+    left: 50%;
+    z-index: $overlay-z-index + 1;
+    max-width: $indicator-max-width;
+    height: 40px;
```

Comment:
I'm not sure you want to mix height and padding when it comes to centering content. I think you'd want either padding and flex with no fixed height, or fixed height and flex without vertical padding, which runs the risk of not handling overflow

Reply:
Gotcha. Vertical padding was calculated off of `line-height: 20px`, but I think I'll use the flex  box without vertical padding

Reply:
Also, the paddings aren't "on the grid" $bdl-grid-unit which is a but surprising coming from a new design